misc/python3/ratchet-passing

```diff
@@ -9,6 +9,9 @@ allmydata.test.test_abbreviate.Abbreviate.test_abbrev_time_year
 allmydata.test.test_abbreviate.Abbreviate.test_parse_space
 allmydata.test.test_abbreviate.Abbreviate.test_space
 allmydata.test.test_abbreviate.Abbreviate.test_time
+allmydata.test.test_backupdb.BackupDB.test_basic
+allmydata.test.test_backupdb.BackupDB.test_upgrade_v1_v2
+allmydata.test.test_backupdb.BackupDB.test_wrong_version
```

Comment:
I was surprised to see these here but here's how I rationalized it.  The ratchet fails if there are failing tests listed as expected to pass _or_ if there are passing tests not listed as expected to pass.

So the changes here caused some tests which previously failed to start passing, even though neither those tests nor the implementations they cover are *explicitly* being ported by this PR.

Does that seem right?